analyzer/analyzer_test.go

```diff
@@ -16,6 +16,13 @@ func TestAnalyzer(t *testing.T) {
 			desc:     "all",
 			patterns: "simple",
 		},
+		{
+			desc:     "all",
```

Comment:
```suggestion
			desc:     "all options",
```

`desc` should be unique.

I also suggest renaming the previous test case "all" to "default".